### ISSUE TITLE
Cleanup the wugbot GitHub repo in preparation for deployment

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/core#18]

**Comment by wugology** [comment]
ELI5 what exactly needs to be done? There are a lot of great ideas floating around but I haven't been to all the meetings and I'm still learning GitHub.

**Comment by dwhieb** [comment]
I've actually done most of this one already. I copied all the projects in the Wugbot organization into the sandbox repo of DLX. I'm just waiting on @amundo to grant me owner access to the Wugbot org so I can delete what I copied. That'll leave just one repo: wugbot.org. And even that should be cleared out, now that I think on it, because we're going to replace it with the new version we're making in DLX.

I don't know if anybody's explained it yet, but our workflow is this:
DLX > Wugbot > wugbot.org

So DLX is the developer / in progress area, and Wugbot is the clean final code for everything on the website. And wugbot.org is literally just going to be a copy of what's in the Wugbot org. Every once in a while we're do a release, and push some finished or updated product from DLX > Wugbot.

Annnnnyway, short answer: Don't worry about this particular issue. If you wanted to start diving into stuff, maybe assign yourself one of the other issues in this milestone. In particular we need someone to do the HTML and css on the homepage, as well as for the About Wugbot page. [Here's](https://github.com/digitallinguistics/core/tree/master/website) the folder with the website files. I've already added the CSS reset and an index.html page. Part of this would include beginning to come up with a wugbot stylesheet. You already gave use the wugbot color! But eventually we'll want a standard page template, standard styling for buttons/menus, that kind of thing. I figured that, once the home page was done, we could copy it and turn it into a page template we could use for other parts of the site. So I also created a folder called [wugbotStyleGuide](https://github.com/digitallinguistics/core/tree/master/wugbotStyleGuide), which includes the beginning of an HTML page illustrating the various UI elements and how they should look. You could definitely be adding to that as you flesh out the design.

I've never created a style guide before, but I bet there are some helpful examples out there.

It would also be great to document our design principles as we go, and to run mockups by the group. So I think Pat prefers lots of white space and bolding, and simple yet pretty, and I tend to agree with him. So in our style guide we might say things like 'prefer lots of white space'.

(What's EL15? I'm so un-cool. I don't know these things.)

**Comment by wugology** [comment]
ELI5 means "explain like i'm five (years old)." As in, I have literally no idea what's going on please help.

I really, really want to help but I don't know how to "assign" myself to a task. And most of the tasks aren't very clearly explained.

**Comment by dwhieb** [comment]
Sorry that's my bad :-/ I'll be better about that in the future.

So I think the perfect one to start with is issue #13 .

**Comment by dwhieb** [comment]
Oh crap you're not even a member of the org yet! That would help. One sec.
